### DISCUSSION TITLE
Tem como fazer o remix carregar os scripts só depois que a página estiver carregada?

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#2245] [Perguntas e Respostas]
https://remix.run/docs/en/v1/api/remix#meta-links-scripts 

Tentei usar async no `<Scripts />`, mas não funciona... tem como fazer algo do tipo?
```html
<script async src="../"></script>
```

Alguém já passou por isso?

**Comment by felipe-heredia** [comment]
Eu acho que não é possível criar um async dentro do html já que ele é estático. Acho que você precisaria de um JS pra fazer a requisição desse outro JS.

**Comment by igorthurow** [comment]
@felquis Fala bro, beleza? Mano, eu não conheço o Remix, até parei pra dar uma lida aqui, achei massa. 

Mesmo não conhecendo o framework, pensei aqui que caso o Remix não tenha essa feature de forma nativa, você pode adicionar um `eventListener` de `DOMContentLoaded` ou `load` e instanciar o `<Script/>` de forma dinâmica baseada num estado controlado por esse listener.
